renderer/src/components/Calendar/Calendar.tsx

```diff
@@ -193,6 +194,11 @@ export function Calendar({
     });
   }, [selectedDate]);
 
+  useEffect(() => {
+    if (timetrackerUserInfo) {
+      trackConnections("timetrackerWeb");
```

Comment:
Create a constant for trackConnections args.
An example `const TRACK_CONNECTIONS = { TIMETRACKER_WEB: 'timetrackerWeb' ... };`